libs/ui-watt/src/lib/components/input/datepicker/+storybook/watt-datepicker-reactive-forms.stories.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { FormControl, ReactiveFormsModule, Validators } from '@angular/forms';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
+import { within, fireEvent } from '@storybook/testing-library';
+
+import { StorybookConfigurationLocalizationModule } from '../../+storybook/configuration-localization/storybook-configuration-localization.module';
+
+import { WattDatepickerComponent } from '../watt-datepicker.component';
+import { WattDatepickerModule } from '../watt-datepicker.module';
+import { WattFormFieldModule } from '../../../form-field/form-field.module';
+import { WattRangeValidators } from '../../shared/validators';
+
+export default {
+  title: 'Components/Datepicker',
+  decorators: [
+    moduleMetadata({
+      imports: [
+        ReactiveFormsModule,
+        WattFormFieldModule,
+        WattDatepickerModule,
+        BrowserAnimationsModule,
+        StorybookConfigurationLocalizationModule.forRoot(),
+      ],
+    }),
+  ],
+  component: WattDatepickerComponent,
+} as Meta;
+
+const template = `
+<watt-form-field>
+  <watt-label>Single date</watt-label>
+  <watt-datepicker [formControl]="exampleFormControlSingle"></watt-datepicker>
+  <watt-error *ngIf="exampleFormControlSingle?.errors?.required">
+      Date is required
+  </watt-error>
+</watt-form-field>
+
+<p>Value: <code>{{exampleFormControlSingle.value | json}}</code></p>
+<p *ngIf="withValidations">Errors: <code>{{exampleFormControlSingle?.errors | json}}</code></p>
+
+<br />
+
+<watt-form-field>
+  <watt-label>Date range</watt-label>
+  <watt-datepicker [formControl]="exampleFormControlRange" range></watt-datepicker>
```

Comment:
issue: in VS Code I get the following error if I simply use the `range` attribute.

![image](https://user-images.githubusercontent.com/1096332/167855895-908ca304-c57c-4e3b-88d8-48489d2c6197.png)

Let's use the "expanded" version, i.e. `[range]="true"` when displaying a date range picker.